bindings/python/multibody/frame.hpp

```diff
@@ -51,12 +51,24 @@ namespace pinocchio
                           "A Plucker coordinate frame related to a parent joint inside a kinematic tree.\n\n",
                           bp::no_init
                          )
+        .def(bp::self == bp::self)
         .def(FramePythonVisitor())
         .def(CopyableVisitor<Frame>())
         .def(PrintableVisitor<Frame>())
+        .def_pickle(Pickle())
         ;
       }
 
+    private:
+      struct Pickle : bp::pickle_suite
```

Comment:
It would be better to rely on the serialization module of Pinocchio for pickling, similar to what it is done for Model.
It is more efficient and easy to do.

Reply:
Yes, I had noticed it, I can certainly do it

Reply:
Done